### PR TITLE
Prep for 0.8.0 release

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,15 +72,6 @@ After the container is started, you will be prompted to sign in to your Azure ac
...
 The default behavior of the extension is to connect to the local Docker daemon. You can connect to a docker-machine instance if you launch Visual Studio Code and have the [Docker environment variables](https://docs.docker.com/machine/reference/env/) set in your environment or through the following VS Code settings: `docker.host`, `docker.certPath`, `docker.tlsVerify`, and `docker.machineName`.
 
-### VS Code remote development
-
-By default, the Docker extension will run against your local VS Code instance. You can change this behavior to run against your remote VS Code instance with the following VS Code setting:
-```json
-"remote.extensionKind": {
-    "ms-azuretools.vscode-docker": "workspace"
-}
-```
```
Comment (PrashanthCorp) [inline README.md]: Is there a scenario where running as a UI extension will help? If so, I'd prefer rewording this section to removing it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+## 0.8.0 - 12 August 2019
+### Added
+* Changed default behavior in VS Code remote environments to run as a "workspace" extension instead of a "UI" extension. See [#954](https://github.com/Microsoft/vscode-docker/issues/954) for more information
+* Added support to debug ASP.NET Core web apps with SSL enabled
```
Comment (EricJizbaMSFT) [inline CHANGELOG.md]: @bwateratmsft I feel like I'm not fully aware of all that you did for this release - is this line good enough?

File: README.md (hunk 1 above)
Comment (EricJizbaMSFT) [reply README.md]: Yeah there's a few scenarios, but not as many as running in workspace. We're adding a few wiki pages - I can re-insert this section with a link to those after we finish writing them:

https://github.com/microsoft/vscode-docker/wiki/Docker-in-a-Dev-Container
https://github.com/microsoft/vscode-docker/wiki/Docker-in-Remote-SSH
https://github.com/microsoft/vscode-docker/wiki/Docker-on-WSL-2

File: CHANGELOG.md (hunk 2 above)
Comment (bwateratmsft) [reply CHANGELOG.md]: Yes, I think it's good. No user changes are needed to their launch.json to get things working so this is a freebie for them.

**Pull request merged**